### ISSUE TITLE
17999

### ISSUE THREAD

**Issue by gnakan** [gnakan/gd-cloud-docs#58]

**Comment by Carla-Johnson** [comment]
The filename and the article contents are completely different topics.

**Comment by gnakan** [comment]
I'm not seeing this file in the repo...

**Comment by cjamesbpcs** [comment]
Is there something I need to do to put it in there, or was it removed?
________________________________

**Comment by cjamesbpcs** [comment]
@Carla-Johnson, I believe Susan is re-working or putting this back together as there was a lil mix up

**Comment by Carla-Johnson** [comment]
OK, I just uploaded it (I think).

-Carla

**Comment by gnakan** [comment]
@Carla-Johnson: Is this one still blocked?

**Comment by Carla-Johnson** [comment]
@gnakan: Yes, still blocked. The document's contents does not match the filename. They're two completely different topics.

**Comment by susankayjordan** [comment]
SO this is actually 18229

**Comment by susankayjordan** [comment]
Changing the article number to match content

**Comment by Carla-Johnson** [comment]
Edited. Please see comments within the HTML.

**Comment by cjamesbpcs** [comment]
This article still has the wrong contents in it.

**Comment by susankayjordan** [comment]
The article has been fixed and content  verified by @cjamesbpcs

**Comment by Carla-Johnson** [comment]
The comments in the doc do not seem to have been addressed. Can you take another look? thanks.

**Comment by susankayjordan** [comment]
Sent the latest version to GitHub

**Comment by susankayjordan** [comment]
Updated content

**Comment by Carla-Johnson** [comment]
The edits still have not been addressed. Conrad, can you take another look?

**Comment by garynakanelua** [comment]
@Carla-Johnson & @susankayjordan : Looks like there are two files for the same article in the repo - https://github.com/gnakan/gd-cloud-docs/blob/master/18229%20-%20Build%20a%20LAMP%20stack%20for%20Python%20-%20CentOS%20Fedora.html and https://github.com/gnakan/gd-cloud-docs/blob/master/18229%20-%20Build%20a%20LAMP%20stack%20for%20Python%20-%20CentOS%2C%20Fedora.html

**Comment by kharise** [comment]
Deleted the duplicate file (that was actually written to Ubuntu/Debian but named Centos/Fedora)

**Comment by kharise** [comment]
Going to need to rewrite to add the Install Python part, and also the other sections have to be modified to enable Python..

Also, removed instruction to run as root.

**Comment by kharise** [comment]
Added in Python section.

Passed testing

Forked off C7, Fedora.

**Comment by wcdean** [comment]
Needed Edits
https://docs.google.com/document/d/19ZLk7VBr668zXb2D2kp520ru7-I2lu4vovfPJlm9FD0/edit?usp=sharing
